### PR TITLE
feat(adp-tooling): add missing adp config

### PR DESCRIPTION
**Pull request by korotkovao**
#1611
https://github.com/SAP/open-ux-tools/issues/1901

Adaptation Tooling Writer for **Migration**

Consume the configuration settings needed in ui5.yaml and `package.json` in order for the`Visual Editor` and `npm run start-editor` to work in BAS.

### CODE REVIEW

File: packages/adp-tooling/test/unit/writer/__snapshots__/index.test.ts.snap [hunk 1]
```diff
@@ -463,10 +491,19 @@ specVersion: \\"3.0\\"
 metadata:
   name: my.test.app
 type: application
+resources:
+  configuration:
+    propertiesFileSourceEncoding: UTF-8
```
Comment (devinea) [inline packages/adp-tooling/test/unit/writer/__snapshots__/index.test.ts.snap:496]: @korotkovao Can you add a test to write safemode to ui5.yaml?

Comment (korotkovao) [reply packages/adp-tooling/test/unit/writer/__snapshots__/index.test.ts.snap:496]: added, thank you @devinea

File: .changeset/gentle-monkeys-flash.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux/adp-tooling': minor
```
Comment (tobiasqueck) [inline .changeset/gentle-monkeys-flash.md:2]: Just make it a patch please. I would consider all the changes fixes and not new features.

**Review by tobiasqueck — CHANGES_REQUESTED** [review]
Please also add `minify` as a excluded task to the build script

**Comment by korotkovao** [comment]
Thanks @tobiasqueck 
>Please also add minify as a excluded task to the build script

Added it to the templates

**Review by docirl — CHANGES_REQUESTED** [review]
Can you address the 5 sonar issues? They are for unused imports. Thanks

**Comment by korotkovao** [comment]
> Can you address the 5 sonar issues? They are for unused imports. Thanks

Thanks @docirl , i missed this.

**Review by devinea — APPROVED** [review]
Code updates and tests appear to cover the migration scenario as expected. 
Changeset ✅

**Review by tobiasqueck — APPROVED** [review]
* changeset ok
* code is clean and easy to understand
* tested locally and the fixes are all corrected in a generated ADP

**Review by docirl — APPROVED** [review]
Comment addressed

**Comment by korotkovao** [comment]
Thank you for the reviews and testing @tobiasqueck @docirl  @devinea

**Pull request merged**